src/dotnet-interactive-vscode-common/src/interactiveClient.ts

```diff
@@ -22,7 +22,7 @@ import {
     KernelEventEnvelope,
     KernelEventEnvelopeObserver,
     KernelEventType,
-    KernelTransport,
+    Connector,
```

Comment:
Can this name be a bit more specific?

Reply:
will align that even more to .net api names